hellohilt/src/main/java/com/airbnb/mvrx/hellohilt/HelloFragment.kt

```diff
@@ -0,0 +1,34 @@
+package com.airbnb.mvrx.hellohilt
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import com.airbnb.mvrx.Fail
+import com.airbnb.mvrx.Loading
+import com.airbnb.mvrx.MvRxView
+import com.airbnb.mvrx.Success
+import com.airbnb.mvrx.Uninitialized
+import com.airbnb.mvrx.fragmentViewModel
+import com.airbnb.mvrx.withState
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.android.synthetic.main.fragment_hello.helloButton
+import kotlinx.android.synthetic.main.fragment_hello.messageTextView
+
+@AndroidEntryPoint
+class HelloFragment : Fragment(R.layout.fragment_hello), MvRxView {
+
+    val viewModel: HelloViewModel by fragmentViewModel()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        helloButton.setOnClickListener { viewModel.sayHello() }
```

Comment:
Can you migrate this to view binding like the sample app? You can copy in [FragmentViewBindingDelegate](https://github.com/airbnb/MvRx/blob/master/sample/src/main/java/com/airbnb/mvrx/sample/utils/FragmentViewBindingDelegate.kt)